### ISSUE TITLE
Radio stream gets aborted after some time

### ISSUE THREAD

**Issue by fbruetting** [gnumdk/lollypop#1302]
### Environment
- Lollypop version: Current Flatpak
- GNOME/GTK+ versions: 3.26.2 / 3.22.25
- GNU/Linux distribution: Fedore 27

### Bug
Since a few weeks, my radio streams get aborted after playing for a while (maybe an hour or so). This happened with different streams. Any idea? I now try to see if I get helpful console output…

**Comment by gnumdk** [comment]
No idea, any other radio listener in the place?

**Comment by fbruetting** [comment]
No, I use it since like ever. But I try to catch something via CLI output…

**Comment by gnumdk** [comment]
You can enable gstreamer debug environment variables.
